style.css

```diff
@@ -0,0 +1,81 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  z-index: 99999999;
+  background-color: #fff;
+}
+
+.welc {
+  color: #172b4f;
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.menu {
+  color: #172b4f;
+  font-size: 12px;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  background-image: url("/images/header-illsutration-mobile@2x.jpg");
```

Comment:
- [ ] As previously said, this URL path doesn't work because you are searching inside the 'images' folder. However, if you check your directory you used uppercased latter for 'Images'. Kindly change the first letter to lowercase.
![image](https://user-images.githubusercontent.com/91486791/219729696-1142c424-4932-478a-843a-5fc6593f6d9f.png)